test/gpu/native/studies/shoc/triadalt1.compopts

```diff
@@ -0,0 +1 @@
+--fast --gpu-block-size=128 result.chpl
```

Comment:
Before this PR we had a single `COMPOPTS` file which applied to all the tests.
This was OK because all the SHOC Triad tests required the same `COMPOPTS`.

Sort needs different `COMPOPTS` and thus we had to split it up and give every test its own `.compopts` file